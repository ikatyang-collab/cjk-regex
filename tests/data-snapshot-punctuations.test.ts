import { snapshotCharset } from './utilities.js'
import * as cjk from '../src/index.js'

snapshotCharset('punctuations', cjk.punctuations())
