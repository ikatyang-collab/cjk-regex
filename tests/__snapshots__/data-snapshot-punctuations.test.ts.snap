// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Snapshot: punctuations 1`] = `
"
12272 (⿰, U+2FF0) - 12292 (〄, U+3004)
12294 (〆, U+3006)
12296 (〈, U+3008) - 12320 (〠, U+3020)
12330 (〪, U+302A) - 12343 (〷, U+3037)
12348 (〼, U+303C) - 12351 (〿, U+303F)
12448 (゠, U+30A0)
12539 (・, U+30FB) - 12540 (ー, U+30FC)
12688 (㆐, U+3190) - 12703 (㆟, U+319F)
12736 (㇀, U+31C0) - 12783 (㇯, U+31EF)
12831 (㈟, U+321F) - 12895 (㉟, U+325F)
12927 (㉿, U+327F) - 13007 (㋏, U+32CF)
13055 (㋿, U+32FF)
13144 (㍘, U+3358) - 13311 (㏿, U+33FF)
65040 (︐, U+FE10) - 65055 (︟, U+FE1F)
65072 (︰, U+FE30) - 65135 (﹯, U+FE6F)
65280 (＀, U+FF00) - 65381 (･, U+FF65)
65392 (ｰ, U+FF70)
65438 (ﾞ, U+FF9E) - 65439 (ﾟ, U+FF9F)
65471 (﾿, U+FFBF) - 65473 (￁, U+FFC1)
65480 (￈, U+FFC8) - 65481 (￉, U+FFC9)
65488 (￐, U+FFD0) - 65489 (￑, U+FFD1)
65496 (￘, U+FFD8) - 65497 (￙, U+FFD9)
65501 (￝, U+FFDD) - 65519 (￯, U+FFEF)
"
`;
